src/cli_plugin/install/settings.js

```diff
@@ -36,11 +36,20 @@ import expiry from 'expiry-js';
 
 import { fromRoot } from '../../core/server/utils';
 
+const LATEST_PLUGIN_BASE_URL =
+  'https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards';
+
 function generateUrls({ version, plugin }) {
-  return [
-    plugin,
-    `https://artifacts.opensearch.org/downloads/kibana-plugins/${plugin}/${plugin}-${version}.zip`,
-  ];
+  return [plugin, generatePluginUrl(version, plugin)];
+}
+
+function generatePluginUrl(version, plugin) {
```

Comment:
Nit: This could have just been in `generateUrls` since that function doesn't do anything anymore 😂

Reply:
I think there is one called `generateUrls` (line 42) right?